base/random.jl

```diff
@@ -1157,6 +1157,25 @@ randn(dims::Integer...) = randn!(Array{Float64}(dims...))
 randn(rng::AbstractRNG, dims::Dims) = randn!(rng, Array{Float64}(dims))
 randn(rng::AbstractRNG, dims::Integer...) = randn!(rng, Array{Float64}(dims...))
 
+## Circularly-polarized complex normally distributed random numbers.
+const SQRT_HALF = 0.7071067811865475
+@inline function randn(rng::AbstractRNG, T::Type{Complex{Float64}})
+    Complex(SQRT_HALF * randn(rng), SQRT_HALF * randn(rng))
+end
+function randn!(rng::AbstractRNG, A::AbstractArray{Complex{Float64}})
```

Comment:
You should be able to just expand the type signature in the existing `randn!` method rather than adding a copy of this method here.